File: FastClean/FastCleanAPI/DTO's/Funcionario/FuncionarioRegister.cs
using FastCleanAPI.DTO_s.Utilizador;
using FastCleanAPI.Models;
using System.ComponentModel.DataAnnotations;

namespace FastCleanAPI.DTO_s.Funcionario
{
    public class FuncionarioRegister : UtilizadorRegister
    {
        [Required]
        public double Preco { get; set; }
        public IFormFile CartaDeConducao { get; set; } 

        public TipoLimpeza TipoLimpeza { get; set; }
        
        public IFormFile HistoricoMedico { get; set; }

        public IFormFile CvFile { get; set; }

        
        public bool Subscricao { get; set; }    

        public virtual List<ReportFuncionario> ListaDeReports { get; set; }

        public virtual List<ReviewFuncionario> ListaDeReviews { get; set;}


        public FuncionarioRegister()
        {
             ListaDeReports = new List<ReportFuncionario>();
             ListaDeReviews = new List<ReviewFuncionario>();
        }
    }
}

File: FastCleanApp/app/src/main/java/com/example/fastclean/Models/Utilizador/Funcionario/FuncionarioRegister.kt
package com.example.fastclean.Models.Utilizador.Funcionario

import com.example.fastclean.Models.Marcacao.Marcacao
import com.example.fastclean.Models.Review.FuncReview
import com.example.fastclean.Models.Utilizador.Morada
import com.example.fastclean.Models.Utilizador.UtilizadorRegister
import java.io.Serializable

class FuncionarioRegister (id: Int,
                           nome: String,
                           password: String,
                           cargo: String,
                           email: String,
                           morada: Morada,
                           contacto: Int,
                           ccfile: String,
                           imagem: String,
                           cadastro: String,
                           listaDeMarcacoes: List<Marcacao>,
                           listaDeChats: String,
                           cartaDeConducao: String,
                           historicoMedico : String,
                           cvFile : String,
                           subscricao : Boolean,
                           listaDeReports : String,
                           listaDeReviews : List<FuncReview>
): UtilizadorRegister(id,
    nome,
    password,
    cargo,
    email,
    morada,
    contacto,
    ccfile,
    imagem,
    cadastro,
    listaDeMarcacoes,
    listaDeChats
) , Serializable {
}